### PR TITLE
add yaqd-fakes

### PR DESCRIPTION
**Pull request by untzag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ksunden** [comment]
Willing to maintain

**Comment by untzag** [comment]
@conda-forge/help-python I need a hint about why my package is failing on Windows :confounded:

**Comment by ksunden** [comment]
@untzag there is no `entry points` section specified in `build`

**Comment by untzag** [comment]
:clap: thanks so much @ksunden I totally missed that! I wonder how it ever passed on Linux?

File: recipes/yaqd-fakes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - yaqd-fake-continuous-hardware = yaqd_fakes._fake_continuous_hardware:FakeContinuousHardware.main
+    - yaqd-fake-discrete-hardware = yaqd_fakes._fake_discrete_hardware:FakeDiscreteHardware.main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - flit
+    - python >=3.7
+    - yaqd-core >=2020.05.1
```
Comment (scopatz) [inline recipes/yaqd-fakes/meta.yaml:26]: Is this really need to create the package?

Comment (untzag) [reply recipes/yaqd-fakes/meta.yaml:26]: yes, [flit](https://flit.readthedocs.io/en/latest/index.html) needs all dependencies at build time

Comment (scopatz) [reply recipes/yaqd-fakes/meta.yaml:26]: That is annoying

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**